packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelForm.tsx

```diff
@@ -126,6 +129,14 @@ export default function ColumnModelForm(props: ColumnModelFormProps) {
     [isJsonSubColumn],
   )
 
+  const [currentMaxSize, setCurrentMaxSize] = useState(
+    (columnModel as ColumnModelFormData).maximumSize ?? '',
+  )
```

Comment:
Why do you need this in state? Can't you just use `(columnModel as ColumnModelFormData).maximumSize`?